portal-api/src/v1/facility-dates/requested-cover-start-date.js

```diff
@@ -34,26 +21,40 @@ const hasAllRequestedCoverStartDateValues = (facility) => {
 };
 
 exports.hasAllRequestedCoverStartDateValues = hasAllRequestedCoverStartDateValues;
-
+/**
+ * Returns facility object with added requestedCoverStartDate property, if the day
+ * month, year are given. This is stored as a UTC timestamp
+ * @param {Object} facility 
+ * @returns {Object}
+ * 
+ * This function has odd behaviour inherited from moment js
```

Comment:
The dependence on this behaviour wants investigating and removing (under a different ticket) if possible